frontend/src/components/MovieCarousel.jsx

```diff
@@ -25,6 +25,9 @@ function MovieCarousel({ type, url }) {
 
   useEffect(() => {
     getMovies();
+    return () => {
+      window.stop();
```

Comment:
I show that, but maybe it would be a better way to use controller abort methode.